index.js

```diff
@@ -14,17 +14,21 @@ const moons = [
 
   You should not need to make any other changes to the code.
 */
-function organizeMoonsByPlanet(moons) {
-  const result = {};
+// LET NOT CONST
+let organizeMoonsByPlanet = (moons) => {
```

Comment:
This function _can_ be declared using `const`—it was only the `closest` below that needed to be declared using `let` (since we were re-assigning it).

You almost never want to reassign a function, object, or array, so they can be (again, _almost_) always declared using `const`.